lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -505,6 +508,21 @@ class DetailsRenderer {
     if (item.selector) element.setAttribute('data-selector', item.selector);
     if (item.snippet) element.setAttribute('data-snippet', item.snippet);
 
+    if (!item.boundingRect || !this._fullPageScreenshot) {
+      return element;
+    }
+
+    const fullPageScreenshot = this._fullPageScreenshot;
+
+    const elementScreenshot = ElementScreenshotRenderer.render(
+      this._dom,
+      this._templateContext,
+      fullPageScreenshot,
+      item.boundingRect,
+      {width: 147, height: 100}
```

Comment:
very magical numbers.  they seem to imply a maxThumbnailWidth/maxThumbnailHeight, right?  I'd prefer they get named that, at least here and inside render()